### PR TITLE
fix broken badge

### PR DESCRIPTION
**Pull request by jukent**
Related: https://github.com/badges/shields/issues/8671

Badge on the Readme is broken. I think this is how to fix it.

### CODE REVIEW

**Comment by jukent** [comment]
The site is failing unrelated to this PR

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Project Pythia Portal
 
-![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/NCAR/pythia-portal/deploy-site/main?logo=github&style=for-the-badge)
+![GitHub Workflow Status (branch)](https://img.shields.io/github/actions/workflow/status/NCAR/pythia-portal/ci.yaml?branch=main)
```
Comment (brian-rose) [inline README.md:3]: This is still pointing to the wrong repo.

I think this is the correct badge link:
```suggestion
[![deploy-site](https://github.com/ProjectPythia/projectpythia.github.io/actions/workflows/ci.yaml/badge.svg?branch=main)](https://github.com/ProjectPythia/projectpythia.github.io/actions/workflows/ci.yaml)
```

**Review by ktyle — APPROVED** [review]

**Comment by brian-rose** [comment]
I just merged #310 so we should probably change the badge to the new `nightly-build` action. I'll open a separate PR for that.

**Comment by brian-rose** [comment]
@jukent ok to close this now that #311 is merged?

**Pull request closed without merging**